src/eeg_series.jl

```diff
@@ -176,7 +176,7 @@ function eeg_topoplot_series!(
 end
 
 """
-function df_timebin(df, Δbin; col_y=:erp, fun=mean, grouping=[])
```

Comment:
the "correct" way to do this is simply to use
```julia
    df_timebin
```
that is 4 spaces - it is somewhere in the docstring documentation in the julia manual

Reply:
I think zou can then drop the backticks + same thing in UnfoldMakiePyMNEExt.jl

Reply:
and rel_to_abs_bbox